frontend/isoko/src/components/business_dashboard/SortReviewsDropdown.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Dropdown, DropdownButton } from 'react-bootstrap';
+
+const StyledDropdownButton = styled(DropdownButton)`
+   background: #fff;
+   color: #000;
+   text-decoration: underline;
+   font-size: 0.5rem;
```

Comment:
Im pretty sure that the styles above this arent doing anything. If so, should be removed